vampire_survivors/src/powerup.py

```
@@ -0,0 +1,7 @@
+import pygame
+
+class Powerup(pygame.sprite.Sprite):
+    def __init__(self, pos, groups):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)